flang/test/Semantics/modfile57.f90

```diff
@@ -0,0 +1,14 @@
+! RUN: %python %S/test_modfile.py %s %flang_fc1
+
+! Cray pointer
+module m
+  integer :: pte
+  pointer(ptr,pte)
```

Comment:
Thank you for the test.  Can you please also add a case where the pointee's shape is defined by the `pointer` statement?

Something like this:
```
integer :: x
pointer(ptrx, x[10])
```

Reply:
Added array